portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    /*response.setContentType("text/html;");
+    response.getWriter().println("<h1>Hello Oyin!</h1>");*/
+
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("How do I sign up to go next year?");
+    comments.add("Is Grace Hopper a real person?");
+    comments.add("Wow! I bet you got a lot of swag!lol ");
+   
+    response.setContentType("application/json;");
+    response.getWriter().println(convertToJsonUsingGson(comments));
+
+  }
+  private String convertToJsonUsingGson(ArrayList comments){
```

Comment:
function name here is kind of ambiguous. Maybe something like convertCommentListToGson()

also space before {

Reply:
Hmm I would argue that this method could actually be renamed to `convertToJson`.
* `usingGson` describes the implementation details of the function, so it shouldn't be in the method name. This method shouldn't be tied down to using gson.
* this method can convert any list of strings into json, so `commentList` feels a little restrictive

Does this function need to take in an ArrayList? Prefer to make parameter types generic - in this case we can use List<String> instead.

This function can also be static since it doesn't use any members of the class.